src/syntax-highlight/syntax-highlight.js

```diff
@@ -39,7 +39,7 @@ export default function syntaxHighlight(diff, afterWordDiff) {
     }
 
     const $diff = $(diff)
-    syntaxHighlightSourceCodeLines($diff)
+    // syntaxHighlightSourceCodeLines($diff)
```

Comment:
i commented to know why we started the process and reapplied it later with afterWordDiff callback, it was another reason of performance issue, some lines lost their token/color syntax after the second highlight

Reply:
2 tests are currently failing because i commented this line, so depends on the answer I will change the tests